src/components/CertificateDropZone/CertificateDropZone.js

```diff
@@ -0,0 +1,152 @@
+import React from "react";
```

Comment:
are all this copy/paste from opencerts  (all this === certificate drop zone)

Reply:
yep, ouch.

Reply:
no worries, just for information =)